test/extension/utils.test.js

```diff
@@ -148,41 +153,46 @@ describe('Test extension utils', () => {
     spy.restore();
   });
 
-  // it('assembleProject', async () => {
-  //   // todo: mock
-  //   const {
-  //     owner, repo, ref, host,
-  //   } = await utils.assembleProject({
-  //     giturl: 'https://github.com/adobe/business-website/tree/main',
-  //   });
-  //   expect(owner).to.equal('adobe');
-  //   expect(repo).to.equal('business-website');
-  //   expect(ref).to.equal('main');
-  //   expect(host).to.equal('business.adobe.com');
-  // });
-
-  // it('addProject', async () => {
-  //   // todo: mock
-  //   const added = await new Promise((resolve) => {
-  //     utils.addProject({
-  //       owner: 'adobe',
-  //       repo: 'business-website',
-  //     }, resolve);
-  //   });
-  //   expect(added).to.be.true;
-  // });
-
-  // it('deleteProject', async () => {
-  //   // todo: mock
-  //   await utils.addProject({
-  //     owner: 'adobe',
-  //     repo: 'business-website',
-  //   });
-  //   const deleted = await new Promise((resolve) => {
-  //     utils.deleteProject(0, resolve);
-  //   });
-  //   expect(deleted).to.be.true;
-  // });
+  it('removeConfig', async () => {
+    const spy = sinon.spy(window.chrome.storage.sync, 'remove');
+    await utils.removeConfig('sync', 'name');
+    expect(spy.calledWith('name')).to.be.true;
+    spy.restore();
```

Comment:
if the expect above fails, the spy is never removed. better to remove all mocks in afterEach or in a try/finally block. 

see https://sinonjs.org/releases/latest/spies/#using-a-spy-to-wrap-all-object-method